assimilation_code/modules/observations/ocean_quantities_mod.f90

```diff
@@ -32,11 +32,53 @@
 !   QTY_PHYTOPLANKTON_BIOMASS
 !   QTY_ALKALINITY
 !   QTY_DISSOLVED_INORGANIC_CARBON
+!   QTY_DISSOLVED_ORGANIC_CARBON
 !   QTY_DISSOLVED_ORGANIC_NITROGEN
 !   QTY_DISSOLVED_ORGANIC_P
 !   QTY_DISSOLVED_INORGANIC_IRON
 !   QTY_SURFACE_CHLOROPHYLL
 !   QTY_LAYER_THICKNESS
+!   QTY_COLUMN_DEPTH
+!   QTY_DISSOLVED_INORGANIC_SIO3
+!   QTY_MESOZOOPLANKTON_CARBON
+!   QTY_MICROZOOPLANKTON_CARBON
+!   QTY_PARAM_AUTOTROPH1_KCO2
```

Comment:
What's the goal of having _PARAM_ in the QTY name?  Just to communicate that it is a parameter?

Do you need a qty for each parameter?  Could you have QTY_PARAMETER?

Reply:
If I remember correctly, I created multiple `QTY_PARAM_...` types in order to facilitate experiments where multiple parameters are being estimated at once. For experiments of this type, we need a way to distinguish one parameter from another, although they do indeed have the same physical properties. But maybe there's some way for us to get by with only one QTY code which I'm not seeing.